gurmukhiutils/unicode.py

```diff
@@ -0,0 +1,448 @@
+from typing import Literal, Match
+
+UNICODE_STANDARDS = Literal["Unicode Consortium", "Sant Lipi"]
```

Comment:
an enum might make more sense here a little easier to compare and more explicit in the code